bottlelib/Data/List/Utils.hs

```diff
@@ -34,7 +34,7 @@ insertAt n x xs = take n xs ++ x : drop n xs
 isLengthAtLeast :: Int -> [a] -> Bool
 isLengthAtLeast n list
     | n <= 0 = True
-    | otherwise = list ^? Lens.ix (n-1) & Lens.has Lens._Just
+    | otherwise = Lens.element (n-1) `Lens.notNullOf` list
```

Comment:
Too bad that `notNullOf = has` - if we have had `has = flip notNullOf` then it **could have been**
```| otherwise = list `Lens.has` Lens.element (n-1)```, which is much more English-like, which is maximum readability :)